astropy/io/fits/hdu/table.py

```diff
@@ -125,7 +125,8 @@ def from_columns(cls, columns, header=None, nrows=0, fill=False,
         coldefs = cls._columns_type(columns)
         data = FITS_rec.from_columns(coldefs, nrows=nrows, fill=fill,
                                      character_as_bytes=character_as_bytes)
-        hdu = cls(data=data, header=header, character_as_bytes=character_as_bytes, **kwargs)
+        hdu = cls(data=data, header=header,
+                  character_as_bytes=character_as_bytes, **kwargs)
```

Comment:
These are the places where it would be nicer to keep the 100 long lines.

Reply:
Almost all of io.fits is 80, so for me it makes more sense to cut a few longer lines that do not fit well with the surrounding code. "A style guide is about consistency", pep8 says. But we can also reformat everything with black or yapf if you prefer ;).

Reply:
no, black is on the blacklist. 

I don't suggest to reformat everything, but rather, gradually let these logically OK lines stay as they are as opposed to be strict 80 long. Also, my aim is to be code consistency within the library rather than keeping the patchwork of modules with their own arbitrary preferences.

Reply:
Arbitrary preferences, really ? io.fits is 20'000 lines of python, and mostly consistent with the 80 chars rule. Consistency within a file itself seems the most important to me, then within a subpackage, then within astropy, because unless you have a solution to reformat 150'000 loc (without automatic tool) I doubt you will achieve this consistency... 
But ok, I will remove the commits that modify line length and just stop worrying about this.